client/cli/src/arg_enums.rs

```diff
@@ -14,10 +14,74 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
+// NOTE: we allow missing docs here because arg_enum! creates the function variants without doc
```

Comment:
maybe we should file a bug with clap about this? Sounds like a reasonable ask...

Reply:
Good point, will do!

Reply:
Issue created: https://github.com/clap-rs/clap/pull/1702

Reply:
It's merged. At some point we will be able to remove that arg_enums module ...